dspace-api/src/main/java/org/dspace/authority/orcid/Orcidv3.java

```diff
@@ -0,0 +1,214 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.authority.orcid;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URLEncoder;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.log4j.Logger;
+import org.dspace.authority.AuthorityValue;
+import org.dspace.authority.SolrAuthorityInterface;
+import org.dspace.external.OrcidRestConnector;
+import org.dspace.external.provider.orcid.xml.XMLtoBio;
+import org.json.JSONObject;
+import org.orcid.jaxb.model.v3.release.common.OrcidIdentifier;
+import org.orcid.jaxb.model.v3.release.record.Person;
+import org.orcid.jaxb.model.v3.release.search.Result;
+
+/**
+ * @author Jonas Van Goolen (jonas at atmire dot com)
+ * This class contains all methods for retrieving "Person" objects calling the ORCID (version 2) endpoints.
+ * Additionally, this can also create AuthorityValues based on these returned Person objects
+ */
+public class Orcidv3 implements SolrAuthorityInterface {
```

Comment:
Could we not rename this class to have a better name instead of Orcidv3, something like Orcidv3SolrAuthorityInterface should make it a bit more clear what this class does.

Reply:
this is aligned with the previous name that was Orcidv2, that said I agree that we can use a better name but please review my general comment about your refactoring suggestion

Reply:
done, named Orcidv3SolrAuthority**Impl** as it is an implementation